src/Schedules.jl

```diff
@@ -188,44 +201,54 @@ struct EveryCalendarDtSchedule{P <: Dates.Period, T <: AbstractFloat} <:
     dt::P
 
     """The `Dates.DateTime` used to convert from simulation time to date."""
-    reference_date::Dates.DateTime
-
-    """Simulation time at the beginning of the simulation. Typically used in restarts."""
-    t_start::T
+    start_date::Dates.DateTime
 
     """
         EveryCalendarDtSchedule(dt::Dates.Period,
-                                reference_date::Dates.DateTime,
-                                date_last::Union{Nothing, Dates.DateTime} = nothing,
-                                t_start = 0)
+                                start_date::Dates.DateTime,
+                                date_last::Union{Nothing, Dates.DateTime} = nothing)
 
     True every time the current date is larger than the previous date this schedule was true + dt.
 
-    The date is computed assuming that `integrator.t` is in seconds and using `reference_date`.
+    The date is computed assuming that `integrator.t` is in seconds and using `start_date`.
     Schematically:
     ```julia
-    date = reference_date + Second(t_start + integrator.t)
+    date = start_date + Second(integrator.t)
     ```
 
-    When `date_last` is `nothing`, `date_last` is computed from `reference_date` and `t_start`.
+    When `date_last` is `nothing`, `date_last` is assumed to be `start_date`.
```

Comment:
Would date_last only not be nothing if we restarted a run?

Reply:
This is a knob that you can use if you want to initialize the schedule to a different value for any reason. It may be used in restarts, but diagnostics across restarts are complex, so we would probably restart from 0 anyway.